storage/aptosdb/src/state_merkle_db.rs

```diff
@@ -78,10 +78,12 @@ impl StateMerkleDb {
         let state_merkle_db_config = rocksdb_configs.state_merkle_db_config;
         // TODO(grao): Currently when this value is set to 0 we disable both caches. This is
         // hacky, need to revisit.
-        //
-        // TODO(grao): Support cache sharding.
         let enable_cache = max_nodes_per_lru_cache_shard > 0;
-        let version_cache = VersionedNodeCache::new();
+        let mut version_caches = HashMap::with_capacity(NUM_STATE_SHARDS + 1);
+        version_caches.insert(None, VersionedNodeCache::new());
```

Comment:
what is the use of None: VersionedNodeCache::new()?

Reply:
For the top levels that are not part of shards (currently only root).